index.html

```diff
@@ -7,6 +7,9 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Movie Explorer</title>
     <link rel="stylesheet" href="styles/styles.scss">
+    
+    <link href="//vjs.zencdn.net/7.10.2/video-js.min.css" rel="stylesheet">
+    <script src="//vjs.zencdn.net/7.10.2/video.min.js"></script>
```

Comment:
You installed the Video.js via NPM, why are you linking the CDN in here?

Reply:
I try something and forgot about it